debian/rules

```diff
@@ -294,22 +294,15 @@ override_dh_auto_install:
 		/usr/share/plymouth/themes/spinner/*				\
 		/usr/bin/unsquashfs						\
 		/sbin/dmsetup							\
-		/usr/lib/snapd/snap-bootstrap					\
-		/usr/lib/snapd/info						\
 		/lib/systemd/systemd-bootchart					\
-		/lib/systemd/system/snapd.recovery-chooser-trigger.service;	\
+		; \
```

Comment:
I think you use emacs. Maybe you can do `C-c C-\` which does `makefile-backslash-region`

Reply:
Good one, I'll use that in the next one